<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplayTimeline</class>
 <widget class="QWidget" name="ReplayTimeline">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget_play" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1071</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color:#323232;
	color:white;
	font-family: &quot;Microsoft YaHei&quot;;
	font-size: 12px;
}
QPushButton {
	background-color:#323232;
	border-width:1;
    padding:4px;  
	color:white;
	border-style:inset;
	border-color: rgb(255, 255, 255);
}
QPushButton:checked {  
    /* 改变背景色   
    background-color:rgb(40,85,20); */
    /* 使文字有一点移动 */  
   /* padding-left:6px;  
    padding-top:6px;  */
	color:black;
	border-color: rgb(255, 255, 255);
	background-color:#ffffff;
}  
</string>
   </property>
   <widget class="QPushButton" name="pushButton_24h">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>11</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>24hr</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2h">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>11</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>2hr</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1h">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>11</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>1hr</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_30min">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>11</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>30min</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget_date" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>130</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(76, 76, 76);</string>
    </property>
    <widget class="QLabel" name="label_date">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>3</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_date">
     <property name="geometry">
      <rect>
       <x>106</x>
       <y>5</y>
       <width>14</width>
       <height>12</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/table_date.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_slow">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/images/replay_slow.png)}
QPushButton:pressed{border-image: url(:/images/replay_slow_p.png)}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_play">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/images/repaly_play.png)}


</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_fast">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/images/replay_fast.png)}
QPushButton:pressed{border-image: url(:/images/replay_fast_p.png)}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_normalimg">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>8</width>
      <height>8</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/replay_normal.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_normal">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Normal Video</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_alarmimg">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>8</width>
      <height>8</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/replay_alarm.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_alarm">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Alarm Video</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_novideoimg">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>8</width>
      <height>8</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/replay_noVideo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_novideo">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Video Loss</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_voice">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/replay_voice.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>20</y>
      <width>140</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
border: 1px solid #323232;
background: #80ffffff;
height:4px;
border-radius: 2px;
}

QSlider::sub-page:horizontal {

background: #476BFD;
border: 1px solid #476BFD;
height:	4px;
border-radius: 2px;

}


QSlider::handle:horizontal 
{
  
  background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5,stop:0.778409 rgba(255, 255, 255, 255));
    width: 10px;
    margin-top: -3px;
    margin-bottom: -3px;
    border-radius: 5px;
}



</string>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_video1">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>10</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/images/repaly_1.png)}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_video4">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>10</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/images/replay_4.png)}



</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
